package.json

```diff
@@ -99,12 +99,14 @@
     "prep": "yarn delete-cache && scripts/gather-credentials.js",
     "start": "yarn prep && gatsby develop",
     "build": "yarn prep && gatsby build --prefix-paths",
+    "postbuild": "yarn lint",
```

Comment:
Added `yarn lint` as `postbuild` so that we can run lint check on GitHub Actions